crates/trie/sparse/src/trie.rs

```diff
@@ -769,15 +778,15 @@ impl<P> RevealedSparseTrie<P> {
         prefix_set: &mut PrefixSet,
         buffers: &mut RlpNodeBuffers,
     ) -> RlpNode {
-        let starting_path = buffers.path_stack.last().map(|item| item.path.clone());
+        let _starting_path = buffers.path_stack.last().map(|item| item.path.clone());
```

Comment:
underscore because starting_path is only used in the trace! macro and in no_std this is a noop, so clippy warns of an unused variable